bin/run_test.py

```diff
@@ -1,31 +1,27 @@
 #!/usr/bin/python
 
 from __future__ import print_function
-import os, sys, subprocess, shutil, json
-from glob import glob
+import os, sys, subprocess, shutil
+
+project_root = os.path.dirname(os.path.dirname(__file__))
+test_utils_dir = os.path.join(project_root, 'test', 'shared')
 
 def single_run(test_project):
-    # load project settings into environment
-    env_file = os.path.join(test_project, 'environment.json')
-    project_env = {}
-    if os.path.exists(env_file):
-        with open(env_file) as f:
-            project_env = json.load(f)
-
-    # run the build
+    # set up an environment that gives access to the test utils
     env = os.environ.copy()
-    project_env = {str(k): str(v) for k, v in project_env.items()} # unicode not allowed in env
-    env.update(project_env)
-    print('Building %s with environment %s' % (test_project, project_env))
-    subprocess.check_call([sys.executable, '-m', 'cibuildwheel', test_project], env=env)
-    wheels = glob('wheelhouse/*.whl')
-    print('%s built successfully. %i wheels built.' % (test_project, len(wheels)))
+    env.update({
+        'PYTHONPATH': test_utils_dir,
```

Comment:
Maybe, just to be sure:
```
pythonpath = test_utils_dir.get('PYTHONPATH', '')
if pythonpath:
    pythonpath += os.pathsep
env.update({
    'PYTHONPATH': pythonpath + test_utils_dir
})
```

(Or something like that; I'm not completely happy with that code)

Reply:
good point. It's a surprisingly annoying bit of code to write, I settled on

```python
    if 'PYTHONPATH' in env:
        env['PYTHONPATH'] += os.pathsep + test_utils_dir
    else:
        env['PYTHONPATH'] = test_utils_dir
```

Reply:
Anything's good, there, indeed :-) I was writing a oneliner, to then think that that's not the kind of code to commit to a project ;-)